projects/giveth/index.js

```diff
@@ -34,26 +42,106 @@ async function stakingXDAI() {
 async function poolXDAI() {
   const balance = await sumUnknownTokens({
     owners: [
-      "0x08ea9f608656A4a775EF73f5B187a2F1AE2ae10e", // GIV / HNY
-      "0x55FF0cef43F0DF88226E9D87D09fA036017F5586", // GIV / ETH
-      "0xB7189A7Ea38FA31210A79fe282AEC5736Ad5fA57", // GIV / XDAI
+      "0xbf945292dc5cbbc8b742083f87d502699cb27414", // GNO - Balancer - Owned By Giveth DAO
+      "0x85dc9beb2571298c9197ec16fafa556a85e41eae", // BRIGTH - Balancer - Owned by Giveth DAO
+      "0x08ea9f608656a4a775ef73f5b187a2f1ae2ae10e", // HNY - Honeyswap - Owned By Giveth DAO
+      "0xdccAa73705dC7457bcfb3dAFEe529B30920e3008", // GNO - Balancer - Not Owned
+      "0xb7189a7ea38fa31210a79fe282aec5736ad5fa57", // xDAI - Honeyswap - Not Owned
+      "0x75594f01da2e4231e16e67f841c307c4df2313d1", // FOX - Honeyswap - Not Owned
+      "0x55ff0cef43f0df88226e9d87d09fa036017f5586", // WETH - Sushiswap - Not owned
     ],
     tokens: [xdaiGIV],
     chain: "xdai",
   });
   return balance;
 }
 
+// ✅ OPTIMISM
+async function stakingOptimism() {
+  const balance = await sumUnknownTokens({
+    owners: ["0x301C739CF6bfb6B47A74878BdEB13f92F13Ae5E7"], // GIvfarm on Optimism
+    tokens: [optimismGIV],
+    chain: "optimism",
+  });
+  return balance;
+}
+
+async function poolOptimism() {
+  const balance = await sumUnknownTokens({
+    owners: [
+      "0xc2ab457e31c224da284df7afda70c39523df4972", // OP - Velodrome - Owned By Giveth DAO
+      "0x165E6DAD9772C8CB44015eDD5bd8b012A84bd276", // USDGLO - Uniswap V3 - Owned By Giveth DAO
+      "0x969e1D236289742C9D36eA1c7124cdDb84397772", // DAI - Uniswap V3 - Owned By Giveth DAO
```

Comment:
this is an uniswap v3 pool, the tvl here belongs to uniswap, not you.


Only if an LP token or nft is deposited by people other than your dao into smart contracts controlled by you, we can count them as tvl, like this contract: https://etherscan.io/address/0x647c509af2a2b2294bb79fce12daec8e7cf938f7